webapp/i18n/en.json

```
@@ -1 +1,3 @@
-{}
+{
+    "plugin.name": "Test Plugin"
```

Can we have this take a param since the caller is supplying an integer like `Test Plugin 1`?

```suggestion
    "plugin.name": "Test Plugin {num}"
```

Example in the webapp:

https://github.com/mattermost/mattermost-webapp/blob/28f2ac8c05323cbd0fa1ab26aec92eb1e2079cef/components/activity_log_modal/components/more_info.tsx#L74

`values` prop supplies the values
```jsx
<FormattedMessage
    id='activity_log.sessionId'
    defaultMessage='Session ID: {id}'
    values={{
        id: currentSession.id,
    }}
/>
```

https://github.com/mattermost/mattermost-webapp/blob/28f2ac8c05323cbd0fa1ab26aec92eb1e2079cef/i18n/en.json#L54
```json
{
   "activity_log.sessionId": "Session ID: {id}"
}
```